docs/5.x/overview.md

```diff
@@ -1,76 +1,89 @@
+Application Bundles may contain "empty", pre-packaged Kubernetes for
+centralized management of Kubernetes resources within an organization or may
+also contain other applications running on Kubernetes. 
 
-0. Prepare your application to run on [Kubernetes](https://k8s.io). If
+The typical cluster lifecycle consists of the following:
+
+0. Prepare your application(s) to run on [Kubernetes](https://k8s.io). If
    you do not have Kubernetes expertise, our Implementation Services team can help.
-1. Package the application into a deployable `tar` tarball ("Application Bundle" or
-   "Bundle").
-2. Publish the Application Bundle for distribution.
-3. Deploy and install the Application Bundle onto Linux based server cluster
-   ("Gravity Cluster" or "Cluster").
-4. Securely connect to any Cluster to monitor health and roll out
-   updates.
+1. Package the applications into a deployable `tar` tarball ("Application Bundle" or
+   "Bundle") using Gravity's `tele` tool.
+2. Publish the Application Bundle for distribution. AWS S3 or any CDN can be used
+   to publish downloadable images. The enterprise edition of Gravity also comes with
+   a web application ("Ops Center") which can be used to publish and manage
+   Application Bundles.
+3. Deploy and install the Application Bundle onto any supported Linux-based
+   infrastructure ("Gravity Cluster" or "Cluster").
+4. Securely connect to any Cluster to monitor health and roll out updates.
 
 ## Packaging
 
-Gravity works only with applications running on Kubernetes. This means the following
-prerequisites apply in order to use Gravity:
+To prepare an Application Bundle for distribution via Gravity, you have to:
+
+* Create Kubernetes resources describing your application(s). [Helm charts](https://helm.sh/) are supported for this.
+* Write an Application Manifest described below.
+* Place all of these files in the same location.
+
+The Application Manifest is required to describe hardware/system requirements
+of your Cluster and to customize the process of creating a new
+Cluster (i.e. the installation of the Cluster).
 
-* The application is packaged into containers.
-* You have Kubernetes resource definitions for application services, pods, etc.
+!!! tip "Gravity Roadmap Tip":
+    Kubernetes community is working on finalizing the cluster API spec. Once it
+    becomes production ready, Gravity developers will be looking into adopting the
+    future standard to replace the application manifest in the future. Meanwhile, 
+    it continues to be the only production-ready method of describing hardware
+    requirements for K8s clusters.
 
-To prepare a Kubernetes application for distribution via Gravity, you have to write
-an Application Manifest. Below is a sample of a simple Application Manifest in YAML format. It follows Kubernetes configuration conventions:
+Below is a simple Application Manifest in YAML format. It follows Kubernetes
```

Comment:
I think it meant to say "sample", not "simple". The manifest below is not simple at all as it include all possible fields :)